packages/api/src/routes/medical/__tests__/mapi/e2e/facility.ts

```diff
@@ -0,0 +1,43 @@
+import { USState, FacilityCreate, Facility } from "@metriport/api-sdk";
+import { faker } from "@faker-js/faker";
+
+export const createFacility: FacilityCreate = {
+  name: faker.word.noun(),
+  npi: "2974324529",
+  active: true,
+  address: {
+    addressLine1: faker.location.streetAddress(),
+    city: faker.location.city(),
+    state: USState.CA,
+    zip: faker.location.zipCode("#####"),
+    country: "USA",
+  },
+};
```

Comment:
Its not necessarily the same because its not using the domain models as the types but rather creating payloads for the medicalApi to use. Let me know what you think